zeno_client/client.py

```diff
@@ -263,8 +263,8 @@ def __init__(self, api_key: str, *, endpoint: str = DEFAULT_BACKEND) -> None:
 
     def create_project(
         self,
-        *,
         name: str,
+        *,
```

Comment:
Made this named because it is for systems as well now. Shouldn't this be unified?

Reply:
IMO it does not hurt to have names as named params but open to go both ways. It might be confusing is one is and the other isn't.

Reply:
Yeah works for me, i'll close.